sigh/spec/runner_spec.rb

```diff
@@ -142,6 +142,17 @@
       end
     end
 
+    describe "#certificates_to_use" do
+      it "list certificates found when there're multiple certificates and not in development" do
+        options = { skip_certificate_verification: true }
+        Sigh.config = FastlaneCore::Configuration.create(Sigh::Options.available_options, options)
+        sigh_stub_spaceship_connect
+
+        certificates_to_use = fake_runner.certificates_to_use
+        expect(certificates_to_use.size).to eq(1)
+      end
+    end
```

Comment:
My first test written in ruby, ever 😅 it looks terrible, but I didn't know how to validate the message thrown by `UI.message` 😬 I thought of extracting the message builder to a separate function, but didn't want to mess around with code much 😅 

Any tips are welcome 😇